# Lab | Inferential statistics

- It is assumed that the mean systolic blood pressure is `μ = 120 mm Hg`. In the Honolulu Heart Study, a sample of `n = 100` people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. 

## 1. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

### 1.1 Set up the hypothesis test.

#### The hypothesis

- Null Hypothesis:  The mean systolic blood pressure of the group, a research conduected by Honolulu Heart Study, **does not have significantly different results** from the regular population. 


- Alternative Hypothesis: The mean systolic blood pressure of the group **has significantly different results** from the regular population. 


### 1.2 Write down all the steps followed for setting up the test.

1. Set the Significance Level
2. Calculate the Test Statistic and Corresponding P-Value
3. Draw a Conclusion

- P-value <= significance level (a) => Reject the null hypothesis in favor of your alternative hypothesis. The result is statistically significant.

- P-value > significance level (a) => Fail to reject the null hypothesis. The result is not statistically significant.

### 1.3 Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

In [24]:
all_mean = 120
hawaii_mean = 130.1
hawaii_std = 21.21
n = 100

# Set the Significance Level
significance_level = 0.05

In [25]:
# Calculate the Test Statistic
test_statistic = (hawaii_mean - all_mean)/(hawaii_std/(n**0.5))
round(test_statistic,5)

4.7619

In [26]:
# Calculate Corresponding P-Value

p_value = 4.9999

#### Conclusion

The p_value(4.9999) is greater than the significance level(4.7619), which means, **it failed to reject the null hypothesis**. 

The mean systolic blood pressure of the group **does not have significantly different results** from the regular population. 

## 2. If you finished the previous question, please go through the code for principal_component_analysis_example provided in the files_for_lab folder .